### PR TITLE
Migrate to AndroidX

### PR DESCRIPTION
**Pull request by B4UGUYS**

### CODE REVIEW

File: android/src/main/java/com/stfalcon/frescoimageviewer/ImageViewer.java [hunk 1]
```diff
@@ -199,7 +202,8 @@ String format(T t) {
 
         private Context context;
         private DataSet<T> dataSet;
-        private @ColorInt int backgroundColor = Color.BLACK;
+        private @ColorInt
+        int backgroundColor = Color.BLACK;
```
Comment (stereodenis) [inline android/src/main/java/com/stfalcon/frescoimageviewer/ImageViewer.java]: ?

File: android/src/main/java/com/stfalcon/frescoimageviewer/ImageViewer.java [hunk 2]
```diff
@@ -149,7 +151,8 @@ public static ImageRequestBuilder createImageRequestBuilder() {
         void onDismiss();
     }
 
-    private @StyleRes int getDialogStyle() {
+    private @StyleRes
+    int getDialogStyle() {
```
Comment (stereodenis) [inline android/src/main/java/com/stfalcon/frescoimageviewer/ImageViewer.java]: ?

File: package.json [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
+        "url": "https://github.com/merryjs/photo-viewer/issues"
+    },
+    "homepage": "https://github.com/merryjs/photo-viewer#readme",
+    "devDependencies": {
+        "@types/prop-types": "^15.5.2",
+        "@types/react": "^16.0.25",
+        "@types/react-native": "^0.50.7",
+        "typescript": "^2.6.1"
+    },
+    "dependencies": {}
```
Comment (stereodenis) [inline package.json]: I think you shouldn't gave your editor config to this repo

File: android/src/main/java/com/stfalcon/frescoimageviewer/ImageViewer.java (hunk 1 above)
Comment (B4UGUYS) [reply android/src/main/java/com/stfalcon/frescoimageviewer/ImageViewer.java]: I think my code formatter is not working there

File: android/src/main/java/com/stfalcon/frescoimageviewer/ImageViewer.java (hunk 2 above)
Comment (B4UGUYS) [reply android/src/main/java/com/stfalcon/frescoimageviewer/ImageViewer.java]: Here the same

**Comment by B4UGUYS** [comment]
Hello guys , just pinging 🙂🙂

**Comment by bang88** [comment]
Thanks for your PR. Did you test it if so I will publish a new version to NPM later.

**Comment by B4UGUYS** [comment]
Yeah @bang88 , I tested it and it is working fine.

**Pull request merged**